Package.swift

```diff
@@ -29,6 +29,9 @@ let package = Package(
             name: "llbuildAnalysis",
             targets: ["llbuildAnalysis"]),
     ],
+    dependencies: [
```

Comment:
We should start respecting the SWIFTCI_USE_LOCAL_DEPS env var so that we can build against a local copy of the dependency in Swift CI

Reply:
Example from swift-driver: https://github.com/swiftlang/swift-driver/blob/6a507035ea6a75dc865cc68a7e69f8f42c176595/Package.swift#L161

Reply:
Good point, added.